examples/api/src/index.ts

```diff
@@ -107,37 +107,60 @@ async function asyncMain() {
       : undefined
   )
 
+  const prisma = new PrismaClient({
```

Comment:
Prisma is now passed down instead of MongoURL to allow for seeding in old initialize function